packages/sourcecred/src/analysis/types.js

```diff
@@ -1,7 +1,13 @@
 // @flow
 
-import {type NodeAddressT, type EdgeAddressT} from "../core/graph";
-import {type EdgeWeight} from "../core/weights";
+import {
+  type NodeAddressT,
+  type EdgeAddressT,
+  NodeAddress,
+  EdgeAddress,
+} from "../core/graph";
+import {type EdgeWeight, edgeWeightParser} from "../core/weights";
+import * as G from "../util/combo";
```

Comment:
why G?

Reply:
just for giggles https://github.com/sourcecred/sourcecred/blob/675f7998693d1fa0c712f589eb3924329203446c/packages/sourcecred/src/analysis/types.js#L82

Reply:
just for giggles https://github.com/sourcecred/sourcecred/pull/3153/files/675f7998693d1fa0c712f589eb3924329203446c#diff-f884d27fbdb018fb62423fd0ec8b782a5c14f22ebdf58c464b79fb605ffc66f3R82